core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -0,0 +1,86 @@
+com.google.common.collect.AbstractSetMultimap
+com.google.common.collect.EmptyImmutableSet
+com.google.common.collect.EmptyImmutableSortedSet
+com.google.common.collect.HashMultimap
+com.google.common.collect.ImmutableList
+com.google.common.collect.ImmutableSet$SerializedForm
+com.google.common.collect.ImmutableSortedSet
+com.google.common.collect.RegularImmutableList
+com.google.common.collect.RegularImmutableSet
+com.google.common.collect.RegularImmutableSortedSet
+com.google.common.collect.SingletonImmutableList
+com.google.common.collect.SingletonImmutableSet
+hudson.maven.MavenInformation
+java.io.File
+java.lang.Boolean
+java.lang.Class
+java.lang.Double
+java.lang.Enum
+java.lang.Float
+java.lang.Integer
+java.lang.Long
+java.lang.Number
+java.lang.Object
+java.lang.StackTraceElement
+java.lang.String
+java.lang.reflect.Proxy
+java.net.URL
+java.security.KeyRep
+java.util.ArrayDeque
+java.util.ArrayList
+java.util.Arrays$ArrayList
+java.util.Collections$EmptyList
+java.util.Collections$EmptySet
+java.util.Collections$SetFromMap
+java.util.Collections$SingletonList
+java.util.Collections$SingletonMap
+java.util.Collections$SingletonSet
+java.util.Collections$UnmodifiableCollection
+java.util.Collections$UnmodifiableList
+java.util.Collections$UnmodifiableRandomAccessList
+java.util.Collections$UnmodifiableSet
+java.util.Date
+java.util.EnumMap
+java.util.GregorianCalendar
+java.util.HashMap
+java.util.HashSet
+java.util.Hashtable
+java.util.LinkedHashMap
+java.util.LinkedHashSet
+java.util.LinkedList
+java.util.Locale
+java.util.Properties
+java.util.RegularEnumSet
+java.util.Stack
+java.util.TreeMap
+java.util.TreeMap$KeySet
+java.util.TreeSet
+java.util.UUID
+java.util.Vector
+java.util.concurrent.ConcurrentHashMap
+java.util.concurrent.ConcurrentSkipListMap
+java.util.concurrent.CopyOnWriteArrayList
+java.util.concurrent.CopyOnWriteArraySet
+java.util.concurrent.CountDownLatch
+java.util.concurrent.CountDownLatch$Sync
+java.util.concurrent.atomic.AtomicBoolean
+java.util.logging.Level
+java.util.logging.LogRecord
+java.util.regex.Pattern
+javaposse.jobdsl.dsl.GeneratedJob
+org.acegisecurity.userdetails.User
+org.apache.commons.fileupload.disk.DiskFileItem
+org.apache.commons.fileupload.util.FileItemHeadersImpl
+org.eclipse.jgit.lib.ObjectId
```

Comment:
why is this defined here and not in the git-client plugin?

Reply:
It could be, of course; ditto a few others classes coming from `job-dsl`, `maven-plugin`, and `workflow-support`. Logistically I just wanted to make sure I was producing a `jenkins.war` that did not introduce regressions to popular plugins, without requiring people to apply recent plugin updates. TBD.